src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,62 @@
+* {
```

Comment:
why is that here, it is supposed to be inside the global scss file. I accept it only once then you refactor it for next sprint